build/dependencies.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <AspNetCoreVersion>$(VersionPrefix)-*</AspNetCoreVersion>
```

Comment:
Swapping the order doesn't work because samples don't import version.props at all. They only import dependencies.props.